app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express")
+const app = express()
+
+const {getTopics} = require('./controllers/topics')
+
+
+//ENDPOINTS
+
+app.get("/api/topics", getTopics)
+
+//ERROR HANDLING
+
+app.all("*", (req, res) => {
+  res.status(400).send({ msg: "Invalid path" });
+});
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+});
+
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
```

Comment:
this isn't technically TDD since your first ticket doesn't necessitate it, but I think I'm fine with you adding in all your error handlers at this point